app/models/state_file_md_intake.rb

```diff
@@ -112,6 +110,9 @@
 #
 class StateFileMdIntake < StateFileBaseIntake
   include MdResidenceCountyConcern
+
+  self.ignored_columns += %w[primary_ssb_amount spouse_ssb_amount]
```

Comment:
this is data that clients self-reported. we can't really "backfill" this data back into the json files, so do we just keep it forever indefinitely?

Reply:
I undid the change to drop the columns, and made the calculator attempt to use them if they're present. I also made them nullable in the db with no default value, so that they should not be `.present?` for intakes created after this change.